lib/twilio/call.ts

```diff
@@ -1356,6 +1356,12 @@ class Call extends EventEmitter {
       messageType: messagetype,
       voiceEventSid: voiceeventsid,
     };
+    this._publisher.info('call-message', 'user-defined-message', {
```

Comment:
'user-defined-message' shouldn't be hardcoded. let's use the `messagetype` variable. update onMessageSent as well.

Reply:
I thought we got rid of the enum for 'user-defined-message'? Should I add the enum back?

Reply:
No. There's a `messagetype` variable at the top. We should use that.